packages/spotlight/src/spotlight.js

```diff
@@ -730,21 +730,27 @@ const Spotlight = (function () {
 		 * @param {String|Node} [elem] The spotlight ID or selector for either a spottable
 		 *  component or a spotlight container, or spottable node. If not supplied, the default
 		 *  container will be focused.
-		 * @param {Object} [containerOption] An optional object containing preferred `enterTo`.
-		 *  It will be passed to the `getTargetByContainer` when `elem` is a container.
+		 * @param {Object} [containerOption] An optional The object including `enterTo` and `toOuterContainer`.
+		 * @param {String} [containerOption.enterTo] It will be passed to the `getTargetByContainer`
+		 *  when `elem` is a container.
+		 * @param {Boolean} [containerOption.toOuterContainer] If the proper target is not found, search one
+		 *  repeatedly to outer container.
 		 * @returns {Boolean} `true` if focus successful, `false` if not.
 		 * @public
 		 */
 		focus: function (elem, containerOption = {}) {
 			let target = elem;
 			let wasContainerId = false;
+			let currentContainerNode;
 
 			if (!elem) {
 				target = getTargetByContainer();
+				currentContainerNode = getContainerNode(getLastContainer());
```

Comment:
If we don't support selector type, let's remove this too. And let's add some docs for the constraints.